docs/DafnyRef/UserGuide.md

```diff
@@ -528,7 +528,39 @@ Hi!
 PASSED
 ```
 
-#### 13.5.1.11. `dafny generate-tests` {#sec-dafny-generate-tests}
+#### 13.5.1.11. `dafny doc` [Experimental] {#sec-dafny-doc}
+
+The `dafny doc` command generates html documentation pages describing the contents of each
+module in a set of files, using the documentation comments in the source files.
+This command is experimental; user feedback on the layout of information and the navigation is welcome.
+
+* The format of the documentation comments is described [here](TODO).
```

Comment:
```suggestion
* The format of the documentation comments is described [here (TODO)](TODO).
```

Reply:
Added the link to the appropriate section